src/components/RecoveryWordsInputField.js

```diff
@@ -0,0 +1,177 @@
+/* global AutoComplete */
+/* global MnemonicPhrase */
+/* global AnimationUtils */
+class RecoveryWordsInputField extends Nimiq.Observable {
+    /**
+     *
+     * @param {number} index
+     */
+    constructor(index) {
+        super();
+
+        this._index = index;
+        /** @type {string} */ this._value = '';
+        this.complete = false;
+
+        this.$ = this._createElements();
+        this._setupAutocomplete();
+    }
+
+    /**
+     * @returns {{ element: HTMLElement, input: HTMLInputElement, placeholder: HTMLDivElement }}
+     */
+    _createElements() {
+        const element = document.createElement('div');
+        element.classList.add('recovery-words-input-field');
+
+        const input = document.createElement('input');
+        input.setAttribute('type', 'text');
+        input.setAttribute('autocorrect', 'off');
+        input.setAttribute('autocapitalize', 'none');
+        input.setAttribute('spellcheck', 'false');
+
+        /** */
+        const setPlaceholder = () => {
+            input.placeholder = `${I18n.translatePhrase('recovery-words-input-field-placeholder')}${this._index + 1}`;
+        };
+        I18n.observer.on(I18n.Events.LANGUAGE_CHANGED, setPlaceholder);
+        setPlaceholder();
+
+        input.addEventListener('keydown', this._onKeydown.bind(this));
+        input.addEventListener('blur', this._onBlur.bind(this));
+        /** @type {HTMLInputElement} */
+        this.$input = input;
```

Comment:
You are setting `this.$input` here, but are also returning it and setting it as `this.$.input`. Same for the placeholder, but not for `element`. Please decide for one approach.

Reply:
:+1: